Gemfile

```diff
@@ -5,20 +5,20 @@ gem "sinatra", "1.3.4"
 gem "rake", "~> 10.5"
 gem "rack", "~> 1.6"
 gem "rest-client", "1.8.0"
-gem "logging", "1.8.1"
-gem 'nokogiri', "1.5.5"
+gem "logging", "2.1.0"
+gem 'nokogiri', "1.6.7.2"
 gem 'whenever', require: false
-gem "slop", "3.4.5"
-gem "sidekiq", "2.13.0"
+gem "slop", "4.3.0"
```

Comment:
Contrary to the PR title you're doing a major upgrade here from `3` to `4`.

Slop is the option parser for things in `bin`. A quick attempt to run something results in an error, so I suspect this is a breaking update.

```
vagrant@development:~/govuk/rummager$ bundle exec bin/health_check --type=suggestions
bin/health_check:34:in `block in run!': undefined method `banner' for #<HealthCheckCLI:0x007f33a9044820> (NoMethodError)
```